### ISSUE TITLE
Book page: Edit, Next, and Previous buttons lacks sufficient focus styling

### ISSUE THREAD

**Issue by jhung** [pressbooks/pressbooks-book#189]
### Prerequisites

* [x] Can you reproduce the behavior you wish to address with [all plugins except Pressbooks deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [x] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/), the latest version of [Pressbooks Book](http://github.com/pressbooks/pressbooks-book/releases/latest/), the latest version of [this book theme](http://github.com/pressbooks/pressbooks-book/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [x] Are you experiencing the issue while using this book theme?
* [x] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks-book) to see if your bug or enhancement is already listed?

### Description

When logged in as a user capable of editing a book, the Edit button located on a book page lacks sufficient focus styling. 

### Steps to Reproduce

1. Log in as a book author / editor.
2. View a book page.
3. Repeatedly press tab until keyboard focus arrives on the Edit, Next, or Previous buttons.

**Expected behavior:**

Expect some sort of perceivable border and style change to indicate that keyboard focus has landed on the Edit, Previous, Next buttons.

**Actual behavior:**

For Edit button, the colour changed from a shade of red to another. Colour alone should not be the sole indicator of status ([See WCAG 2.0 1.4.1 Use of Colour](https://www.w3.org/TR/2008/REC-WCAG20-20081211/#visual-audio-contrast-without-color)).

The Next and Previous buttons, a faint white border appears which is hard to see.
